site/src/docs/components/footer/code.mdx

```diff
@@ -90,7 +100,12 @@ import { Footer } from 'hds-react';
         <Link label="Item" />
         <Link label="Item" />
```

Comment:
Links are not showing, they should be `Footer.Link`. This was my bad from my previous renaming components PR.